Controller/CustomItem/SaveController.php

```diff
@@ -136,18 +136,24 @@ public function saveAction(int $objectId, ?int $itemId = null): Response
         $form->handleRequest($request);
 
         if ($form->isValid()) {
-            $this->customItemModel->save($customItem);
+            $itemCount = $this->customItemModel->getRepository()->count(['customObject' => $this->customObjectModel->fetchEntity($objectId)->getId()]);
```

Comment:
This is inefficient way how to get count. This should work too:
```suggestion
            $itemCount = $this->customItemModel->getRepository()->count(['customObject' => $objectId]);
```
And if not then getting a reference from entity manager is also more efficient as it does not need to fetch anything from the database.

Reply:
Changed the approach completely as suggested in the aforementioned comment